esmvaltool/cmorizers/obs/download_scripts/download_cds_satellite_soil_moisture_monthly.py

```diff
@@ -0,0 +1,167 @@
+"""
+Download cds-satellite-soil-moisture.
+
+Script to download cds-satellite-soil-moisture
+from the Climate Data Store(CDS).
+"""
+
+import argparse
+import os
+import subprocess
+import tarfile
+
+import yaml
+
+import cdsapi
+
+
+def download_cds_satellite_soil_moisture():
```

Comment:
```suggestion
def download_cds_satellite_sm():
```
Name too long (Codacy issue).